data/json/furniture.json

```
@@ -1309,7 +1309,7 @@
     "type": "furniture",
     "id": "f_dryer",
     "name": "dryer",
-    "description": "Dry your clothes!",
+    "description": "Dry your clothes!  Would be what you'd do if electricity was running.",
```

You may want to add air quotes:

```suggestion
    "description": "'Dry your clothes!' would be what you'd do if electricity was running.",
```